coalib/misc/Exceptions.py

```diff
@@ -20,10 +15,10 @@ def get_exitcode(exception, log_printer=None):
         exitcode = exception.code
     elif isinstance(exception, VersionConflict):
         log_message = Constants.VERSION_CONFLICT_MESSAGE % str(exception.req)
-        log_printer.log_exception(log_message, exception)
+        LogPrinterMixin().log_exception(log_message, exception)
```

Comment:
If we can merge the two log items into one, we can use ``logging.info(msg, exc_info=True)`` to output the log msg including the traceback, provided that a custom Formatter for exceptions was set up. However, if we want to stick to two log items, we need to define a helper function, which does the same as the LogPrinterMixin.log_exception method. Please implement this functionality as a function and not as a Class with a single method.